metric-anomaly-data-model/src/main/java/rule/source/RuleSource.java

```diff
@@ -1,4 +1,4 @@
-package org.hypertrace.alert.engine.metric.anomaly.task.manager.rule.source;
+package org.hypertrace.alert.engine.metric.anomaly.data.model.rule.source;
```

Comment:
make this a generic interface to read a json file, currently the terminology used is very close to reading an event condition

Reply:
package name is not matching package path